/bayes_scene_est:
  ros__parameters:
    
    callback_period_sec: 1.0
    scene_labels: ['indoor', 'outdoor', 'transportation']
    scene_prior: [.35,.35,.3]

    sensor_names: ['clip']

    clip:
      obs_labels: ['indoor', 'outdoor', 'transportation']
      sensor_model_coeffs: [.7, .1, .2, .1, .8, .1, .25, .25, .5]
      topic: 'clip_scene_category'

